src/tasks/LibraryBuilder/Templates/CMakeLists.txt.template

```diff
@@ -35,8 +34,9 @@ add_library(
 if(TARGETS_ANDROID)
     set(MOBILE_SYSTEM_LIBS
         libz.so
-        log    
+        log
     )
+    add_definitions(-DHOST_ANDROID=1)
 else()
```

Comment:
Should we be more precise on the platforms we support? And have a else case triggering an error with unsupported platforms?

Reply:
We have a guard in the LibraryBuilder task that checks for the platforms TargetOS. Right now its `"android" or "ios" or "tvos" or "maccatalyst" => true`. I can throw a `send_message(FATAL_ERROR...)` in the else guard as well.